source/UberRides/UberButton.swift

```diff
@@ -45,34 +45,38 @@ import UIKit
     }
     
     override public var highlighted: Bool {
-        // Change colors when button is highlighted
         didSet {
-            var color: UberButtonColor
-            switch colorStyle {
-            case .Black:
-                color = highlighted ? .BlackHighlighted : .UberBlack
-            case .White:
-                color = highlighted ? .WhiteHighlighted : .UberWhite
-            }
-            backgroundColor = ColorUtil.uberUIColor(color)
+            changeColors()
         }
     }
     
-    /// Set color scheme, default is black background with white font.
+    private func changeColors() {
```

Comment:
Also, all the variable declarations should come first, so move this function below them.